bin/index.js

```diff
@@ -22,7 +27,7 @@ function help () {
 }
 
 function htmlpdf (source, destination) {
-  var html = fs.readFileSync(source, 'utf8')
+  var html = DOMPurify.sanitize(fs.readFileSync(source, 'utf8'))
```

Comment:
this won't help anything regardong security. I wouldn't take jsdom into the project as dependency. 
the option which sets the flag is enough. Let's release that as v3 then. 
I guess we can can skip a release in v2 as it won't help anything.

Reply:
I mean we should check whether we can limit the origins of urls (content security policy maybe?). That would be much cleaner. In some cases remote content is desired.

Reply:
True, normally there should be a restriction to the source or format of origins or urls in the config before even been able to read the file?

Some like 
```
const config = {
          allowedFolders:  ['/tmp','render'],
         allowedDomains: ['domain1.tld','domain2']
}
```

this way hackers will hit their heads into the wall daily before breakfast 🥞 🍳

Reply:
I think this would be supported using `page.onResourceRequested`: https://phantomjs.org/api/webpage/handler/on-resource-requested.html

Reply:
wow. spot on! then we just match the options in the config with 

```

  var match = requestData.url.match(/wordfamily.js/g); or includes? 
const allowed = config.allowedFolders.contains(requestData.url.split('/')[0])// a bit hacky but you get the idea?
if(!allowed) networkRequest.abort()// rekt

```

Reply:
So glad y'all are figuring this out. Cause I don't really know what is going on. I just want that npm audit vulnerability to go away.

Reply:
well this is easy to fix just the incentive to is the issues sometimes :D I can write a good fix for it tho before it goes even to phantomjs